commcare_connect/microplanning/migrations/0007_remove_workareagroup_opportunity_access.py

```
@@ -0,0 +1,16 @@
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("microplanning", "0005_workareaexpectedvisitcountworkareagroupopportunityaccessevent_and_more"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="workareagroup",
+            name="opportunity_access",
+        ),
+    ]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)